### DISCUSSION TITLE
[Help] Digital Ocean Appwrite deploy problems

### DISCUSSION THREAD

**Discussion by Kappyh** [appwrite/appwrite#3111] [Q&A] [answered]
Hi,
I have a droplet with a appwrite application, installed by the marketplace.
Im facing the issue in any deploy I try to do in the application:
"Docker Error: WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap."
I already tried resize the droplet ram to 4GB, but did not solve the problem.
Someone already had this error with functions deploy?

[Already opened a ticket on digital ocean help center]

**Comment by Kappyh** [comment, marked as answer]
Found the solution.
-Access the droplet console;
-Type:
`nano /etc/default/grub`
- In the grub file, include the following command:
`GRUB_CMDLINE_LINUX="cgroup_enable=memory swapaccount=1"`
- Save the file and restart grub with the command:
`sudo update-grub`
- After that, restart the droplet.


This solved my problem...

**Reply by eldadfux** [reply]
@lohanidamodar should we consider adding this fix to the repo?

**Reply by BechtelCanDoIt** [reply]
This solved my problem too!!!
